### PR TITLE
Add update task commands and functions

### PR DESCRIPTION
**Pull request by VladimirDan**

### CODE REVIEW

File: taskManager.js [hunk 1]
```diff
@@ -13,11 +13,22 @@ function addTask(title, description, deadline) {
...
 
-function deleteTask(id) {
-    dataStorage.deleteTask(id);
+function deleteTask(title) {
+    dataStorage.deleteTask(title);
+  }
+
+function editTask(taskTitle, updatedTask) {
+    dataStorage.updateTask(taskTitle, updatedTask);
+}
```
Comment (vladyatsuk) [inline taskManager.js:22]: Consider renaming editTask to updateTask or rename the updateTask function in dataStorage.js

File: tasks.json
Comment (vladyatsuk) [inline tasks.json:1]: Should we commit changes in this file?

File: index.js [hunk 2]
```diff
@@ -27,9 +27,28 @@ function main() {
             taskManager.deleteTask(deleteTaskTitle);
             break;
 
-    default:
-        console.log('Unknown command');
-    }
+        case 'editTask':
```
Comment (vladyatsuk) [inline index.js:30]: Consider changing the name to 'edit'. This also applies to 'addTask'.

File: tasks.json
Comment (VladimirDan) [reply tasks.json:1]: no

**Pull request closed without merging**